github/repos_traffic_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestRepositoriesService_ListReferrers(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/1/2/traffic/popular/referrers", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: as a matter of convention, we pretty much always use "o" and "r" for placeholder values to represent "owner" and "repo".  So, "/repos/o/r/traffic/popular/referrers"